### PR TITLE
fix: mkrepo main for wingcloud deployment

### PR DESCRIPTION
**Pull request by polamoros**

### CODE REVIEW

File: .mkrepo/main.w [hunk 1]
```diff
@@ -29,7 +29,11 @@ for file in fs.readdir(".") {
   libs.push(lib);
 }
 
-readme.update(libs.copy());
+try {
+  readme.update(libs.copy());
```
Comment (eladb) [inline .mkrepo/main.w:33]: Why is this needed?

**Review by eladb — CHANGES_REQUESTED** [review]

File: .mkrepo/main.w (hunk 1 above)
Comment (polamoros) [reply .mkrepo/main.w:33]: I was trying it out cause I'm getting this error on wingcloud

image

https://wing.cloud/polamoros/winglibs/logs/main
